roles/deploy_user/molecule/default/verify.yml

```diff
@@ -1,7 +1,8 @@
 ---
 - name: Verify
   hosts: all
-  check_mode: yes
+  vars:
+    app_name: molecule_test
```

Comment:
idk what the best practice is for this, but I noticed myself doing this same thing (i.e. duplicating `vars` for tests across `converge.yml`, `verify.yml`, etc). I ended up making a `test.yml` file in the top-level `vars/` directory for the role (which is for "extra" vars) so that i could import it in test playbooks and get rid of the duplication. lmk if that makes sense to you; happy to explain more

Reply:
yes, I wondered about that. I'll try it